### ISSUE TITLE
Kubernetes fails to install

### ISSUE THREAD

**Issue by jtfogarty** [equinix/terraform-metal-multiarch-k8s#73]
I have the below configured in the terraform.tfvars
```
auth_token = "set correctly"
project_id = "set correctly
count_arm = 0
count_x86 = 3
secrets_encryption = false
```
The servers spin up but Kubernetes does not seem to install correctly.

**Comment by jmarhee** [comment]
Hi @jtfogarty,

In trying to replicate this, I found the following error:

```
error execution phase preflight: [preflight] Some fatal errors occurred:
	[ERROR SystemVerification]: could not unmarshal the JSON output of 'docker info':
WARNING: Error loading config file: .dockercfg: $HOME is not defined
```

May I have you check`/var/log/cloud-init-output.log` from the controller node for comparison? 

I am testing a couple of changes to the user-data for the controller pool user-data, but would like to be sure that you're running into the same issue that I am. 

Thanks.

**Comment by jmarhee** [comment]
When I ran the module again using the linked changes in that branch, I was able to spinup the cluster without issue:

```
kubectl --kubeconfig=/etc/kubernetes/admin.conf get nodes
NAME                                      STATUS   ROLES    AGE   VERSION
jmarhee-lab-controller-primary   Ready    master   11m   v1.19.0
jmarhee-lab-x86-blue-01            Ready    <none>   44s   v1.19.0
```
so, if the issue in your logs matches my own (or when you re-run it with the above branch as a reference), and it resolves the issue, let me know.

**Comment by jtfogarty** [comment]
Hi @jmarhee 
I'm seeing the same error in the log
```
error execution phase preflight: [preflight] Some fatal errors occurred:
        [ERROR SystemVerification]: could not unmarshal the JSON output of 'docker info':
WARNING: Error loading config file: .dockercfg: $HOME is not defined
{"ID":"RCF7:WVVD:X6KJ:3RZ7:4WID:L7ZD:IBSD:BWML:ST5L:HGDI:Q2KR:P4MR","Containers":0,"ContainersRunning":0,"ContainersPaused":0,"ContainersStopped":0,"Images":0,"Driver":"overlay2","DriverStatus":[["Backing Filesystem","extfs"],["Supports d_type","true"],["Native Overlay Diff","true"]],"Plugins":{"Volume":["local"],"Network":["bridge","host","ipvlan","macvlan","null","overlay"],"Authorization":null,"Log":["awslogs","fluentd","gcplogs","gelf","journald","json-file","local","logentries","splunk","syslog"]},"MemoryLimit":true,"SwapLimit":false,"KernelMemory":true,"KernelMemoryTCP":true,"CpuCfsPeriod":true,"CpuCfsQuota":true,"CPUShares":true,"CPUSet":true,"PidsLimit":true,"IPv4Forwarding":true,"BridgeNfIptables":true,"BridgeNfIp6tables":true,"Debug":false,"NFd":22,"OomKillDisable":true,"NGoroutines":34,"SystemTime":"2021-04-24T17:08:03.238141112Z","LoggingDriver":"json-file","CgroupDriver":"systemd","CgroupVersion":"1","NEventsListener":0,"KernelVersion":"5.4.0-52-generic","OperatingSystem":"Ubuntu 18.04.5 LTS","OSVersion":"18.04","OSType":"linux","Architecture":"x86_64","IndexServerAddress":"https://index.docker.io/v1/","RegistryConfig":{"AllowNondistributableArtifactsCIDRs":[],"AllowNondistributableArtifactsHostnames":[],"InsecureRegistryCIDRs":["127.0.0.0/8"],"IndexConfigs":{"docker.io":{"Name":"docker.io","Mirrors":[],"Secure":true,"Official":true}},"Mirrors":[]},"NCPU":16,"MemTotal":33570779136,"GenericResources":null,"DockerRootDir":"/var/lib/docker","HttpProxy":"","HttpsProxy":"","NoProxy":"","Name":"metal-multiarch-k8s-controller-primary","Labels":[],"ExperimentalBuild":false,"ServerVersion":"20.10.2","Runtimes":{"io.containerd.runc.v2":{"path":"runc"},"io.containerd.runtime.v1.linux":{"path":"runc"},"runc":{"path":"runc"}},"DefaultRuntime":"runc","Swarm":{"NodeID":"","NodeAddr":"","LocalNodeState":"inactive","ControlAvailable":false,"Error":"","RemoteManagers":null},"LiveRestoreEnabled":false,"Isolation":"","InitBinary":"docker-init","ContainerdCommit":{"ID":"","Expected":""},"RuncCommit":{"ID":"","Expected":""},"InitCommit":{"ID":"","Expected":""},"SecurityOptions":["name=apparmor","name=seccomp,profile=default"],"Warnings":["WARNING: No swap limit support","WARNING: No blkio weight support","WARNING: No blkio weight_device support"],"ClientInfo":{"Debug":false,"Context":"default","Plugins":[],"Warnings":null}}
```

**Comment by jmarhee** [comment]
Thank you for reporting and confirming— the PR linked above should address this change.

**Comment by jmarhee** [comment]
Addressed in #75
